src/Namespace.ts

```diff
@@ -47,6 +47,21 @@ declare global {
       /** Map TMX awards (undefined if the map was never fetched from TMX) */
       awards?: number
     }
+    /** TM Server Challenge return type */
+    export interface ServerMap {
+      Name: string,
```

Comment:
camelcase??

Reply:
This interface was created to be compatible with what client calls return.